modules/local/starfusion/build/main.nf

```diff
@@ -0,0 +1,41 @@
+process STARFUSION_BUILD {
+    tag 'star-fusion'
+
+    conda (params.enable_conda ? "bioconda::dfam=3.3 bioconda::hmmer=3.3.2 bioconda::star-fusion=1.10.0 bioconda::trinity=date.2011_11_2 bioconda::samtools=1.9 bioconda::star=2.7.8a" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "docker.io/trinityctat/starfusion:1.10.1"
```

Comment:
is this going to work if it's docker container? will singularity know to convert it?

Reply:
Yes, singularity just runs the docker container